src/Web3.elm

```diff
@@ -66,7 +67,7 @@ sha3 val =
     toTask
         { func = "sha3"
         , args = Encode.list [ Encode.string val ]
-        , expect = expectString
+        , expect = expectJson keccakDecoder
```

Comment:
Adding slightly more complex decoders will result in a trivial increase in compute cycles.